src/actions/index.js

```diff
@@ -73,10 +74,12 @@ export const SAVE_PROJECT_SETTINGS_FINISH = 'SAVE_PROJECT_SETTINGS_FINISH';
 //
 export const addProject = (
   project: ProjectInternal,
+  projectType: ProjectType,
   isOnboardingCompleted: boolean
 ) => ({
   type: ADD_PROJECT,
   project,
+  projectType,
```

Comment:
It might be good to actually store this in the model, I can imagine us wanting to put it in the UI at some point. But yeah, for now I just need it for analytics